### ISSUE TITLE
Random Error

### ISSUE THREAD

**Issue by Shaybs** [lensapp/lens#2430]
TypeError: Cannot read property 'containers' of undefined
    at Vm.getContainers (http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7831460)
    at http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:8210664
    at http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7440232
    at http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7439845
    at ce (http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:12079)
    at e.track (http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:23443)
    at http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7439821
    at http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7439658
    at G (http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7439795)
    at http://eb5b9213-81fc-48c1-8705-edd086240efc.localhost:49723/build/Lens.js:2:7440211

**Comment by jim-docker** [comment]
Can you give more details? Was this on startup?

**Comment by jmpalomar** [comment]
Same here, it happen to me in normal operation so I closed Lens but now is on startup opening data from an specific namespace, the last I was browsing and there is no way to go to main cluster screen.

Component stack:
`at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527687
    at div
    at div
    at div
    at UD (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:8259603)
    at div
    at RF (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:8288282)
    at div
    at div
    at xF (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:8294453)
    at footer
    at div
    at IU (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:8498861)
    at AA (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7676789)
    at t (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:221963)
    at kU (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:8500365)`

Error stack:
`TypeError: Cannot read property 'containers' of undefined
    at jm.getContainers (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7891201)
    at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:8278420
    at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527708
    at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527321
    at ce (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:12079)
    at e.track (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:23443)
    at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527297
    at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527134
    at G (http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527271)
    at http://554437da-fc1a-4fae-b36f-39dfa8db6b01.localhost:64965/build/Lens.js:2:7527687`

**Comment by Shaybs** [comment]
Yes, it was on startup. Just happened randomly. Stopped after a computer restart.

**Comment by Shaybs** [comment]
Sorry for the late reply, had a busy week.

**Comment by kalihman** [comment]
I'm having same issues randomly, sometimes resolved after restart or not resolved at all. Any idea why this is happening???
```
Error stack:

TypeError: Cannot read property 'containers' of undefined
    at Of.getContainers (http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7899747)
    at http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:8292941
    at http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7553411
    at http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7553024
    at ce (http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:12079)
    at e.track (http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:23443)
    at http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7553000
    at http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7552837
    at G (http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7552974)
    at http://ddbdff2e-b650-4187-8755-4c783122761f.localhost:58821/build/Lens.js:2:7553390
```

**Comment by Nokel81** [comment]
Fixed by #3361
